service/src/integrationTest/kotlin/fi/espoo/evaka/varda/VardaDecisionsIntegrationTest.kt

```diff
@@ -577,11 +577,11 @@ class VardaDecisionsIntegrationTest : FullApplicationTest() {
     }
 
     @Test
-    fun `a decision is full day with more than 25 hours`() {
+    fun `a decision is full day with 25 or more hours`() {
```

Comment:
Should we leave the 25.5 test case and add a new 25.0 test case?

Reply:
Yeah why not, tests are a little bit clearer that way. Fixed.